Android/res/values-es/arrays.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
@jandrop this resource is identical to the one in `values/arrays.xml`, so let's remove it.
Also if strings within the `values/arrays.xml` need to be translated, let's extract them to the `strings.xml` resource, and translate them there.